### DISCUSSION TITLE
Phonon scattering is so different

### DISCUSSION THREAD

**Discussion by caolz** [mesonepigreco/CellConstructor#105] [Q&A] [answered]
Firstly I trained a neural network potential, after that I used tutorial 8: Thermal conductivity calculations with the SSCHA, got the final_dyn and saved it in phononpy format.
![dispersion](https://github.com/user-attachments/assets/13f36ecc-6189-40ff-83ea-80d16deeaf4d)
This diagram is the one I drew using tutorial 1 to read final_dyn, which is basically the same as VASP using phonopy.
![image](https://github.com/user-attachments/assets/49dbaf87-478a-4c13-a064-1b32ec2b127e)
I drew this picture using FORCE_CONSTANTS saved as a phonopy format, and tried VASP format vs QE format, but the difference is too great as well. 

I wonder if the developers haven't seen anything similar?

**Comment by mesonepigreco** [comment, marked as answer]
Hi, phonopy export is broken unfortunately. There is a workaround implemented by @DjordjeDangic here 

https://github.com/SSCHAcode/CellConstructor/issues/87

Check if this works for you
